cabal2nix/test/golden-test-cases/highlighting-kate.nix.golden

```diff
@@ -18,5 +18,5 @@ mkDerivation {
   ];
   homepage = "http://github.com/jgm/highlighting-kate";
   description = "Syntax highlighting";
-  license = "GPL";
+  license = lib.licenses.gpl2Plus;
```

Comment:
Upstream still uses `GPL`, and has `gpl2Only`, I think: https://github.com/jgm/highlighting-kate/blob/master/LICENSE